### PR TITLE
Fix `BLOB` conversion in `parquet_metadata`

### PR DESCRIPTION
**Pull request by lnkuiper**
The data was correct but not retrieved correctly from the file in the `parquet_metadata` function.

### CODE REVIEW

**Review by Mytherin — COMMENTED** [review]
Thanks for the PR! LGTM - one minor comment

File: extension/parquet/parquet_statistics.cpp [hunk 1]
```diff
@@ -149,13 +149,12 @@ Value ParquetStatisticsUtils::ConvertValue(const LogicalType &type, const duckdb
...
 	}
-	case LogicalType::VARCHAR:
-	case LogicalType::BLOB:
-		if (Value::StringIsValid(stats)) {
-			return Value(stats);
-		} else {
+	case LogicalTypeId::VARCHAR:
+	case LogicalTypeId::BLOB:
+		if (type.id() == LogicalTypeId::BLOB || !Value::StringIsValid(stats)) {
 			return Value(Blob::ToString(string_t(stats)));
```
[the remark this answers is not in the document]
Comment (lnkuiper) [reply extension/parquet/parquet_statistics.cpp:155]: This has the same behavior as before for `VARCHAR`, I've just inverted the `if` (to make the code more readable).

Before:
```c++
		if (Value::StringIsValid(stats)) {
			return Value(stats);
		} else {
			return Value(Blob::ToString(string_t(stats)));
		}
```

This PR:
```c++
		if (type.id() == LogicalTypeId::BLOB || !Value::StringIsValid(stats)) {
			return Value(Blob::ToString(string_t(stats)));
		}
		return Value(stats);
```

If the string is invalid, we do the blob logic (like before, but now also if the type should be `BLOB`). The return type of the `Value` is always `VARCHAR`, as we're creating it with a `std::string` input.

[the remark this answers is not in the document]
Comment (lnkuiper) [reply extension/parquet/parquet_statistics.cpp:155]: Although a test seems to be failing so I probably missed something

[the remark this answers is not in the document]
Comment (Mytherin) [reply extension/parquet/parquet_statistics.cpp:155]: Ah yeah, the diff made it look like there was change in behavior - thanks

**Pull request merged**